ports/raspberrypi/Makefile

```diff
@@ -160,7 +160,7 @@ CFLAGS += $(OPTIMIZATION_FLAGS)
 CFLAGS += $(CFLAGS_CYW43)
 #Debugging/Optimization
 ifeq ($(DEBUG), 1)
-  CFLAGS += -ggdb3 -O3
+  CFLAGS += -ggdb3 -O1
```

Comment:
We would usually use `-Og`, but you said that didn't work. if `-O1` works, let's leave it and add a comment about what was tried and what didn't work.